recipe/meta.yaml

```diff
@@ -2,39 +2,64 @@
 {% set version = "0.9.0" %}
 
 package:
-  name: {{ name|lower }}
+  name: openff-toolkit-split
```

Comment:
Nice catch @jaimergp - I assume this name is only a placeholder and nothing named `openff-toolkit-split` will be built?

Reply:
Yes, since `outputs` is the only thing we are using (I _think_ we could use the main level recipe and then outputs too), it doesn't matter. Log doesn't show any `-split` package being built either.